src/payment/data/sagas.test.js

```diff
@@ -103,6 +102,23 @@ describe('saga tests', () => {
   }
 
   describe('handleFetchBasket', () => {
+    it('should bail if the basket is processing', async () => {
```

Comment:
This test proves that we exit `catchBasketError` right away if the basket is processing.